src/platforms/dotnet/common/compatibility.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Compatibility
+sidebar_order: 50
+description: "Learn more about which version the SDK is compatible with."
+---
+
+According to this [compatibility table](https://docs.microsoft.com/en-us/dotnet/standard/net-standard), the package is compatible with the following versions or newer:
+
+- .NET Framework 4.6.1
+- .NET Core 2.0
+- Mono 5.4
+- Xamarin.Android 8.0
+- Xamarin.Mac 3.8
+- Xamarin.iOS 10.14
+- Universal Windows Platform 10.0.16299
+
+<Note>
+
+If you're using an older version of .NET Framework or Mono, [our legacy SDK](/clients/csharp/) supports .NET Framework as early as 3.5.
+
+</Note>
+
+## Unit Testing
+
+We run our unit/integration tests against:
```

Comment:
I phrase it like this a few years back on the .NET repo but I wonder if we can reword this a bit nicer?

```suggestion
Sentry validates the SDK functionality by running its test suit in the following platforms:
```

Reply:
How about "Sentry validates our SDK functionality running a test suite against the following platforms"